### ISSUE TITLE
Suddenly can't login to certain project anymore

### ISSUE THREAD

**Issue by peXed** [directus/api#1882]
# Bug Report

## Steps to Reproduce

1. Try to login
2. Look at the network console

## Expected Behavior
Backend is loaded.

## Actual Behavior
"Server Error" gets shown and the API call to: https://mydirectusinstallation.com/users/me?fields[0]=id&fields[1]=avatar.*&fields[2]=email&fields[3]=first_name&fields[4]=last_name&fields[5]=locale&fields[6]=role.*&fields[7]=last_page&fields[8]=theme

Ouputs an 401 error. The response is:
`{
  "error": {
    "code": 3,
    "message": "Unauthorized request",
    "class": "Directus\\Exception\\UnauthorizedException",
    "file": "\/var\/www\/vhosts\/httpdocs\/directus\/src\/core\/Directus\/Services\/UsersService.php",
    "line": 152
  }
} <br/ >
  <b> Notice </b>:  Trying to access array offset on value of type null in <b>/var/www/vhosts/ httpdocs/directus/src/core/Directus/Services/ItemsService.php </b> on line <b>557</b> <br/>
`

## Other Context & Screenshots
I did not change anything. The error appeared over night as it seems.
Other projects on the same install are still working perfectly. Only the one project seems broken.

## Technical Details

- Device: Linux Server
- OS: -
- Web Server: -
- PHP Version: 7.4.3
- Database: -
- Directus Release Version: v8.6.2
- Install Method: cloned `master` branch

**Comment by peXed** [comment]
@rijkvanzanten My server is set to the timezone "Europe/Berlin". But should that cookie expiration not be calculated with the timezone of the user?

**Comment by rijkvanzanten** [comment]
Yeah just trying to wrap my head around what's going on and how to resolve it moving forward 🙂

**Comment by reitsma** [comment]
I got the same issues after upgrading from 7.0.* to 8.8.1. Changing the auto signout  from 60 (original value) to 180 fixed the problem. Timezone of the server was Europe/Berlin, user Europe/Amsterdam. Changing user to Europe/Berlin did not fix it (wouldn't expect it too).

Observation of 3 successive calls when logging in with the admin client:
/authenticate call returns cookie with 1 hour slack (correct)
The next call to /_/ set the cookie with 1 hour less (as if it is interpreting the UTC time as Amsterdam time, 1 hour difference).
Then another call to /_/ the cookie has vanished, because it has expired.
After that a server error occurs (in Apache error log), and the response is unauthorized. The error in the log is  
Trying to access array offset on value of type null in /var/www/directus/src/core/Directus/Services/ItemsService.php

So, there are two issues: The null issue from ItemsService.php, and the cookie expiration time juggling.
